tests/core/test_parsing.py

```diff
@@ -234,60 +237,63 @@ def test_parse_notes(page, start, end):
     assert notes.endswith(end)
 
 
-@pytest.mark.parametrize('page,start,end', (
```

Comment:
Are you deleting 10 tests here and replacing them with only 5? I think the trend should be to increase test coverage. We're going to keep needing to update the parser for new variations on the page formats, and it's very likely that will break the parsing process for some old variations.

Reply:
When I went through these tests, it looked like some use cases where duplicated, so I was able to knock them down to half. Again, I don't want to test the page parsing here, but the specific parsing functions.

Reply:
Remember when you were surprised to hear me say I thought the parser was brittle...my code for finding the Notes field was part of what I was talking about. How about if you keep the single-function unit tests you created, but you also keep the old tests that didn't get converted to unit tests, at least until there's something to replace them with that serves exactly the same purpose?

Reply:
We could, but I found a lot of things to redo in our tests, so the full effort will probably cover this. We can formalize it with an issue if you want.